lib/api.es6.js

```diff
@@ -50,6 +47,102 @@ const login = (apiOptions, username, pass) => {
   });
 };
 
+const register = (apiOptions, username, password, email, newsletter) => {
+  return new Promise((r, x) => {
```

Comment:
I know it's a tad bit more verbose, but it'd be nice for these to be `(resolve, reject)`. It's slightly less overhead for the initiated and a lot friendlier for any ES6 newbies stumbling upon this code.